app/models/feedback.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Feedback
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  RATINGS = {
+    5 => 'Very satisfied',
+    4 => 'Satisfied',
+    3 => 'Neither satisfied nor dissatisfied',
+    2 => 'Dissatisfied',
+    1 => 'Very dissatisfied'
+  }.freeze
+
+  attr_accessor :comment, :rating, :email
+  validates :comment, presence: true
+  validates :rating, inclusion: { in: RATINGS.keys.map(&:to_s) }
```

Comment:
**not a blocker** needs a non-default message ideally

Reply:
👍 Added a non-default validation message for rating